### PR TITLE
Cybereason v2.1.17

### PR DESCRIPTION
**Pull request by shabina-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 1]
```diff
@@ -1939,6 +1944,74 @@ def get_machine_details_command(client: Client, args: dict):
...
+        outputs = []
+        malop_output = {
+            'GUID': guid,
+            'Link': link,
+            'CreationTime': creation_time,
+            'LastUpdateTime': malop_last_update_time,
+            'Status': management_status,
+            'InvolvedHash': involved_hashes
+        }
+        outputs.append(malop_output)
```
Comment (jbabazadeh) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py:1987]: I think this lines need to be inside the for loop?

**Review by jbabazadeh — COMMENTED** [review]
Hi @shabina-metron ,
Thank you for your contribution!
Good work :)
Please see my comment

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py (hunk 1 above)
Comment (shabina-metron) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason.py]: Yes, you are correct. We have modified the code and pushed it again. Thanks.

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 2]
```diff
@@ -1939,6 +1944,74 @@ def get_machine_details_command(client: Client, args: dict):
...
+            creation_time = single_malop.get("creationTime", "")
+            malop_last_update_time = single_malop.get("lastUpdateTime", "")
+            management_status = single_malop.get("investigationStatus", "")
+            involved_hashes = single_malop.get("rootCauseElementHashes", [])
+            if single_malop["isEdr"]:
+                link = SERVER + '/#/malop/' + guid
+            else:
+                link = SERVER + '/#/detection-malop/' + guid
+
+            outputs = []
```
Comment (jbabazadeh) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py:1978]: please move the `outputs = []` outside the for loop

Comment (shabina-metron) [reply Packs/Cybereason/Integrations/Cybereason/Cybereason.py]: Yes, I have updated the code.

**Review by jbabazadeh — COMMENTED** [review]
Hi @shabina-metron , the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

**Comment by shabina-metron** [comment]
Hi,

Thanks for your response. I'll reach out to you on DFIR regarding Demo.

Thanks

**Review by jbabazadeh — APPROVED** [review]

**Comment by jbabazadeh** [comment]
Hi @shabina-metron 
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**